R/task.R

```diff
@@ -384,6 +384,81 @@ task_log_fetch <- function(id, root) {
 }
 
 
+status_to_logical <- function(status) {
+  switch(status,
+         submitted = NA,
+         running = NA,
+         success = TRUE,
+         failure = FALSE,
+         cancelled = FALSE,
+         cli::cli_abort("Unhandled status '{status}'"))
+}
+
```

Comment:
Maybe better naming of this function, to include that success -> TRUE  ?
Is `success_status_as_logical` too wordy?
`is_status_success` maybe?
`final_status_sucess` perhaps hints that if the thing isn't final, it returns NA...

(Only used internally and very locally, so no big deal)

Reply:
sure, went with `final_status_to_logical` here